azd-hooks/postprovision.sh

```diff
@@ -64,6 +63,45 @@ az postgres flexible-server firewall-rule create \
 az postgres flexible-server parameter set --name max_prepared_transactions --value 10 -g ${RESOURCE_GROUP_NAME} --server-name ${DB_RESOURCE_NAME}
 az postgres flexible-server restart -g ${RESOURCE_GROUP_NAME} --name ${DB_RESOURCE_NAME}
 
+az cognitiveservices account create \
+    --name ${AZURE_OPENAI_NAME} \
+    --resource-group ${RESOURCE_GROUP_NAME} \
+    --location ${LOCATION} \
+    --kind OpenAI \
+    --custom-domain $AZURE_OPENAI_NAME \
+    --sku s0
+
+resourceId=$(az cognitiveservices account show \
+    --resource-group ${RESOURCE_GROUP_NAME} \
+    --name ${AZURE_OPENAI_NAME} \
+    --query id --output tsv | tr -d '\r')
+
+az resource update \
+    --ids ${resourceId} \
+    --set properties.networkAcls="{'defaultAction':'Allow', 'ipRules':[],'virtualNetworkRules':[]}"
+
+az cognitiveservices account deployment create \
+    --name ${AZURE_OPENAI_NAME} \
```

Comment:
I have my OpenAI capabilities in my Visual Studio Enterprise entitlement, but I want to deploy Liberty on AKS using the `WAS_MSFT` subscription. Can you modify the system to accommodate this requirement?

Reply:
Hello @edburns ,I just found maybe I misunderstood your comments.
What do you mean "  I want to deploy Liberty on AKS using the WAS_MSFT subscription"?
So you want to provision openai resources with `subscriptionA`, but want to deploy the liberty on aks with `subscriptionB`?

Reply:
Hello @edburns ,I just found maybe I misunderstood your comments.
What do you mean "  I want to deploy Liberty on AKS using the WAS_MSFT subscription"?
So you want to provision openai resources with `subscriptionA`, but want to deploy the liberty on aks with `subscriptionB`?

Reply:
Yes, that is correct. I observe that is what I must do when I use my Visual Studio Enterprise OpenAI entitlement, but want to demo stuff from the WAS-MSFT subscription. For example, in this video: https://www.youtube.com/watch?v=Jcbp0D4qM0k .

Reply:
Hello @edburns , check the updates, https://github.com/Azure/azure-dev/issues/4428